tools-local/init-build.proj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <Target Name="Build">
-    <CallTarget Targets="UnpackTarballs;BuildXPlatTasks;AcquireDarc;FetchSources;WriteSourceLinkMetadata;ExtractToolPackage;GenerateRootFs;ApplyPatches;PoisonPrebuiltPackages" />
+    <CallTarget Targets="UnpackTarballs;BuildXPlatTasks;AcquireDarc;FetchSources;WriteSourceLinkMetadata;ExtractToolPackage;GenerateRootFs;ApplyPatches;UpdateBuildToolFramework;PoisonPrebuiltPackages" />
```

Comment:
I would prefer `UpdateBuildToolFramework` be like `UpdateGlobalJsonVersions` and `UpdateNuGetConfig`, with `BeforeTargets="Build"`--to group it with the other repeatable and common replacements vs. the one-time repo-specific patches. I think it'll make it easier to keep track of, in particular make it easier to apply this replacement when trying to do an incremental builds with some custom patch changes.